sql/core/src/test/resources/sql-tests/results/cte-recursion.sql.out

```diff
@@ -325,10 +325,13 @@ SELECT * FROM t
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.sql.catalyst.ExtendedAnalysisException
+org.apache.spark.SparkException
```

Comment:
can we turn this test into a valid one? For now it produces infinite result and can't be supported by Spark